v1.1/admin-ui-overview.md

```diff
@@ -2,6 +2,7 @@
 title: Admin UI Overview
 summary: Use the Admin UI to monitor and optimize cluster performance.
 toc: false
+key: explore-the-admin-ui.html
```

Comment:
Can you add this front-matter field to [`CONTRIBUTING.md`](https://github.com/cockroachdb/docs/blob/master/CONTRIBUTING.md#pages) with a short explanation of what it does?